crates/ruma-events-macros/src/event_enum.rs

```diff
@@ -637,10 +637,9 @@ fn generate_redacted_fields(
         let name = Ident::new(name, Span::call_site());
 
         if name == "unsigned" {
+            // TODO: make constructor
```

Comment:
I don't understand. What does this mean?

Reply:
Ugh I put it there so I could flip back to the spot I needed but should have removed.